utils/diff_output_utils.go

```diff
@@ -0,0 +1,66 @@
+package utils
+
+type DiffResult interface {
+	GetStruct() DiffResult
+	OutputText(diffType string) error
+}
+
+type MultiPackageDiffResult struct {
```

Comment:
nit: Did MultiVersionPackageDiffResult lose the "Version" part intentionally when it became MultiPackageAnalyzeResult? Also is SingleVersionDiffResult now SinglePackageAnalyzeResult? These names seem significantly less clear to me since the real crux of MultiVersion is that it supports multiple versions of the same package. Also SinglePackageAnalyzeResult seems like it would be an analysis of one package, rather than an analysis of all packages, given that each exists with a single version in the image.